composer.json

```diff
@@ -23,21 +23,19 @@
     },
     "require": {
         "php": ">=5.6.0",
-        "doctrine/common": "~2.6",
-        "doctrine/dbal": "~2.5",
-        "symfony/config": "~3.2",
-        "symfony/dependency-injection": "~3.2",
-        "symfony/doctrine-bridge": "~3.2",
-        "symfony/form": "~3.2",
-        "symfony/framework-bundle": "~3.2",
-        "symfony/http-kernel": "~3.2",
-        "symfony/validator": "~3.2",
+        "doctrine/orm": "~2.5",
+        "symfony/config": "3.4.*",
+        "symfony/dependency-injection": "3.4.*",
+        "symfony/framework-bundle": "3.4.*",
+        "symfony/http-kernel": "3.4.*",
+        "symfony/validator": "3.4.*",
         "twig/twig": "~1.30 || ~2.0"
     },
     "require-dev": {
+        "symfony/form": "3.4.*",
+        "symfony/doctrine-bridge": "3.4.*",
         "phpunit/phpunit": "5.7.*",
-        "squizlabs/php_codesniffer": "~2.7",
```

Comment:
Currently the build fails on `/home/travis/.travis/job_stages: line 57: ./vendor/bin/phpcs: No such file or directory`. Well of course `vendor/bin/phpcs` doesn't exist after you removed code sniffer. Should I remove the command from travis or add code sniffer back?